src/aws-provider/aws-organization-writer.ts

```diff
@@ -735,22 +760,22 @@ export class AwsOrganizationWriter {
                 Type: 'Account',
                 Tags: {},
                 SupportLevel: 'basic',
-                PartitionAccountId: accountCreationStatus.GovCloudAccountId,
+                PartitionId: accountCreationStatus.GovCloudAccountId,
```

Comment:
is there a compelling reason to change PartitionAccountId to PartitionId? my guess is that this is the change you referred to as "possibly breaking the account binding". If there is a compelling (or less compelling reason) it might make sense to split this in another PR.

Reply:
My thinking was that 1. There are 3+ types of "GovCloud" partitions, and 2. The changes I am making would make it very simply to allow, instead of mirroring a GovCloud partition, mirroring any partition, including another commercial partition. Because of that, and because of adding the partitionId to the binding for all objects, I don't want to specify what resource the partitionId is set for. Additionally, any future updates may include a resource that we want to explicitly set the "partitionId" for, and this affords those new features to be in line with this convention. 

Anywhere resource could have a physicalId and partitionId. I like that better than all resources have a physicalId, but some may have a partitionId, partitionAccountId, partitionXyzId, etc... Not a major issue, but forcing that in the organization.yml file would be a breaking change. I suggest we add a deprecation notice and a catch for GovCloudAccountId for the organization.yml file, and we can change that to PartitionId. Additionally, in the PartitionAccountUpdateTasks, we should catch the bindings set to PartitionAccountId, and change that PartitionId, automatically. If we do that, the change will not be breaking.